.github/workflows/CI.yml

```diff
@@ -68,46 +56,28 @@ jobs:
           - "1.6"
         os:
           - ubuntu-latest
-
     steps:
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
-
-      - uses: julia-actions/setup-julia@0b9b1d2cd24245f151902702d8e73b3f6b910014
+      - uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@v2
         with:
           version: ${{ matrix.version }}
-
-      - uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353
-        env:
-          cache-name: cache-artifacts
-        with:
-          path: ~/.julia/artifacts
-          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
-          restore-keys: |
-            ${{ runner.os }}-test-${{ env.cache-name }}-
-            ${{ runner.os }}-test-
-            ${{ runner.os }}-
-
-      - name: Install dependencies
-        run: julia --color=yes --project -e "using Pkg; Pkg.instantiate()"
-        
+      - uses: julia-actions/cache@v2
+      - uses: julia-actions/julia-buildpkg@v1
       - run: rm -f versions.json
-
       - name: Build versions.json
         run: |
           using VersionsJSONUtil
-
           VersionsJSONUtil.main("versions.json")
         shell: julia --project {0}
-
       - name: Validate versions.json against schema
         run: npx -p ajv-cli@3.3.0 ajv -s schema.json -d versions.json
-
       - name: Upload versions.json as workflow artifact
-        uses: actions/upload-artifact@c24449f33cd45d4826c6702db7e49f7cdb9b551d
+        uses: actions/upload-artifact@v4
         with:
           name: versions
           path: versions.json
           if-no-files-found: error
+          overwrite: true
```

Comment:
Why is this change necessary?

Reply:
Probably isn't, was for debugging

Reply:
Let's try reverting this change for now, and see if everything still works?